### DISCUSSION TITLE
Funny mistake in build script

### DISCUSSION THREAD

**Discussion by SuchAFuriousDeath** [obhq/obliteration#818] [General]
@ultimaweapon , I don't know where else to put this, but you made a mistake in the build script and it currently kills my Rust analyzer.

![image](https://github.com/obhq/obliteration/assets/48620541/cf2b8b6d-2b7a-4d58-ab9c-f68caa78ca10)

**Comment by SuchAFuriousDeath** [comment]
![image](https://github.com/obhq/obliteration/assets/48620541/3a035428-cef1-41a4-bca0-b29751a87abb)

**Comment by SuchAFuriousDeath** [comment]
(Fix incoming in the next PR)

**Comment by ultimaweapon** [comment]
It is intentional, not a mistake. You need to upgrade Rust to 1.77. The single : is deprecated now. See https://doc.rust-lang.org/cargo/reference/build-scripts.html for more detail.

**Comment by SuchAFuriousDeath** [comment]
:flushed:
